Assets/Plugins/Source/Editor/ConfigEditors/IConfigEditor.cs

```diff
@@ -22,17 +22,28 @@
 
 namespace PlayEveryWare.EpicOnlineServices.Editor
 {
+    using System;
     using System.Threading.Tasks;
 
     // Interface for allowing adding additional config files to the Config editor
-    public interface IConfigEditor
+    public interface IConfigEditor : IDisposable
     {
         /// <summary>
         /// Returns the human-readable labelText for the section of configuration.
         /// </summary>
         /// <returns>String representing the section this config controls.</returns>
         string GetLabelText();
 
+        /// <summary>
+        /// Affect the expansion of the ConfigEditor.
+        /// </summary>
+        public void Expand();
```

Comment:
nit: It feels odd to mark these as public, but not the others. As far as I understand for interface, `public` is always implied.

Reply:
This comment feels like maybe it should be, "expands the ConfigEditor"? What does it mean to "affect the expansion"?

Reply:
Reasonable not on the comment - likewise the access modifier being specified. I default to setting the access modifier because of other languages in which you can have protected or private interface functions (at least I think that's why I do it - might just be I'm used to specifying it whenever I'm writing a function).

In any case, I removed the `public` access modifiers, and changed the comments in both the interface `IConfigEditor`, and in `ConfigEditor`.